src/Avatar/StaticAvatarResolver.php

```diff
@@ -11,19 +11,22 @@
 
 namespace Nucleos\UserAdminBundle\Avatar;
 
+use Symfony\Component\Asset\Packages;
 use Symfony\Component\Security\Core\User\UserInterface;
 
 final class StaticAvatarResolver implements AvatarResolver
 {
+    private Packages $packages;
     private string $defaultAvatar;
 
-    public function __construct(string $defaultAvatar)
+    public function __construct(Packages $packages, string $defaultAvatar)
```

Comment:
Can you please switch the arguments and make the `Packages` dependency nullable. This will help to make this change BC.

Reply:
so it should be nullable (also i did run cs-fixer to check the style)